src/DynamoCoreWpf/UI/Themes/Modern/DynamoModern.xaml

```diff
@@ -6480,7 +6480,7 @@
         <Setter Property="BorderBrush" Value="{StaticResource DarkMidGreyBrush}" />
         <Setter Property="ColumnWidth" Value="Auto" />
         <Setter Property="GridLinesVisibility" Value="Vertical" />
-        <Setter Property="VerticalGridLinesBrush" Value="{StaticResource DarkMidGreyBrush}" />
+        <Setter Property="VerticalGridLinesBrush" Value="#646464" />
```

Comment:
Hmm this looks like backward to me?

Reply:
I updated this to match the design we have on Figma. It meant that I needed to add a new color to the swatch of colors we used so far. https://www.figma.com/design/MJh76m7bbafE3sjwLUTZhZ/Dynamo-package-manager-update?node-id=2397-31482&node-type=canvas&t=OCvkjGxUxDFeO1qW-0

I pushed a new commit making it a resource, which I should have done to begin with, apologies @QilongTang .